client/components/Pipeline/PartnerAssigner/PartnerAssigner.tsx

```diff
@@ -17,6 +18,10 @@ interface PartnerAssignerState {
   show: boolean;
 }
 
+const AddImage = styled(Image)`
+  cursor: pointer;
+`;
+
```

Comment:
Hover state would be nice @albertzhidong

Reply:
done